File: modulo_intermediario/modulo.js
const fs = require('fs');
const path = require('path');

require('cross-fetch/polyfill');

pessoas= [
    {nome: 'Ana', idade: 20},
    {nome: 'Bia', idade: 25},
    {nome: 'Carlos', idade: 30}
]

const saveTo = path.resolve(__dirname, "aquivo-javascript.json")
fetch('https://jsonplaceholder.typicode.com/users')
    .then(res => res.json())
    .then((json)=>{
        fs.writeFileSync(saveTo, JSON.stringify(json, null, 2));
    });

File: modulo_intermediario/modulo.py
import json
import os

pessoas = [
    {"nome": 'Ana', "idade": 20},
    {"nome": 'Bia', "idade": 25},
    {"nome": 'Carlos', "idade": 30}
]

BASE_DIR = os.path.dirname(os.path.abspath(__file__))
CAMINHO_ARQUIVO = os.path.join(BASE_DIR, 'pessoas.json')

with open(CAMINHO_ARQUIVO, 'w', encoding='utf-8') as arquivo:
    json.dump(pessoas, arquivo, indent=2)
